apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/R4Server.java

```diff
@@ -0,0 +1,131 @@
+    setServerName(CAPABILITIES_SERVER_NAME);
+
+    /*
+     * Read in some of the project metadata from a Maven-filtered properties
+     * file, which ensures that it's always up to date.
+     */
+    Properties projectProps = new Properties();
+    try (InputStream projectPropsStream =
+        Thread.currentThread()
+            .getContextClassLoader()
+            .getResourceAsStream("project.properties"); ) {
+      projectProps.load(projectPropsStream);
+
+      setImplementationDescription(projectProps.getProperty("project.id"));
+      setServerVersion(projectProps.getProperty("project.version"));
+    } catch (IOException e) {
+      throw new UncheckedIOException(e);
+    }
+
+    // Lightly customize the capability provider to set publisher name.
+    ServerCapabilityStatementProvider capabilityStatementProvider =
+        new ServerCapabilityStatementProvider(this);
+    capabilityStatementProvider.setPublisher(CAPABILITIES_PUBLISHER);
+    setServerConformanceProvider(capabilityStatementProvider);
+  }
+
+  /** @see ca.uhn.fhir.rest.server.RestfulServer#initialize() */
+  @SuppressWarnings("unchecked")
+  @Override
+  protected void initialize() throws ServletException {
+    /*
+     * Grab the application's Spring WebApplicationContext from the web
+     * container. We can use this to retrieve beans (and anything that needs
+     * Spring injection/autowiring, e.g. anything that accesses the DB, must
+     * be a bean).
+     */
+    WebApplicationContext springContext = ContextLoaderListener.getCurrentWebApplicationContext();
+
+    // Each IResourceProvider adds support for a specific FHIR resource.
+    List<IResourceProvider> resourceProviders =
+        springContext.getBean(SpringConfiguration.BLUEBUTTON_R4_RESOURCE_PROVIDERS, List.class);
+    setResourceProviders(resourceProviders);
+
+    /*
+     * Each "plain" provider has one or more annotated methods that provides
+     * support for non-resource-type methods, such as transaction, and
+     * global history.
+     */
+    List<Object> plainProviders = new ArrayList<>();
+    setPlainProviders(plainProviders);
+
+    /*
+     * Register the HAPI server interceptors that have been configured in
+     * Spring.
+     */
+    Collection<IServerInterceptor> hapiInterceptors =
+        springContext.getBeansOfType(IServerInterceptor.class).values();
+    for (IServerInterceptor hapiInterceptor : hapiInterceptors) {
+      this.registerInterceptor(hapiInterceptor);
+    }
+    /*
+     * Enable CORS.
+     */
+    CorsConfiguration config = new CorsConfiguration();
+    CorsInterceptor corsInterceptor = new CorsInterceptor(config);
+    config.addAllowedHeader("Accept");
+    config.addAllowedHeader("Content-Type");
+    config.addAllowedOrigin("*");
+    config.addExposedHeader("Location");
+    config.addExposedHeader("Content-Location");
+    config.setAllowedMethods(Arrays.asList("GET", "POST", "PUT", "DELETE", "OPTIONS"));
```

Comment:
Another useful suggestion from @karlmdavis , pulled from previous PR: 

> Personally, I'd disagree: this should be modified in both RestfulServer implementations. It's entirely possible that we will never touch this code again: if we don't clean things up as we encounter them, who will? We should always strive to leave things a little better than we found them.
> 
> In this specific case, I don't see why a backend API like BFD needs CORS headers at all? I'd ask in #bfd-users and see if anyone objects to us removing it.

Reply:
CORS Headers removed since BFD is a backend API.